src/Symfony/Component/TypeInfo/Type/TemplateType.php

```diff
@@ -27,33 +25,18 @@ final class TemplateType extends Type
 {
     public function __construct(
         private readonly string $name,
-        private readonly Type $bound,
+        private readonly Type $boundType,
     ) {
     }
 
-    public function getBaseType(): BuiltinType|ObjectType
-    {
-        throw new LogicException(\sprintf('Cannot get base type on "%s" template type.', $this));
-    }
-
-    public function isA(TypeIdentifier|string $subject): bool
-    {
-        return false;
-    }
-
     public function getName(): string
     {
         return $this->name;
     }
 
-    public function getBound(): Type
```

Comment:
is the renaming of this method really worth a BC break ? Experimental components allow BC breaks outside major versions, but we still try to maintain BC as much as possible to make upgrades easier.

Reply:
I didn't know that we had to be that careful with experimental components, I reverted that, thanks